src/libraries/System.Net.WebProxy/src/System/Net/WebProxy.cs

```diff
@@ -68,7 +68,14 @@ public string[] BypassList
             get { return _bypassList != null ? (string[])_bypassList.ToArray(typeof(string)) : Array.Empty<string>(); }
             set
             {
-                _bypassList = new ArrayList(value);
+                if (value == null)
+                {
+                    _bypassList = null;
+                }
+                else
+                {
+                    _bypassList = new ArrayList(value);
+                }
```

Comment:
Does explicitly setting `null` helps the linker in some way, or can it be omitted?

```suggestion
                if (value != null)
                {
                    _bypassList = new ArrayList(value);
                }
```

Reply:
Ah if value was not null before and we set it to null, then it will retain the old value without the null branch. (ignore me) :)